mappings/net/minecraft/client/render/entity/EntityRenderer.mapping

```diff
@@ -12,9 +14,13 @@ CLASS net/minecraft/class_897 net/minecraft/client/render/entity/EntityRenderer
 		ARG 1 entity
 		ARG 2 tickDelta
 	METHOD method_3921 hasLabel (Lnet/minecraft/class_1297;)Z
+		COMMENT Determines whether the passed entity should render with a nameplate above its head.
+		COMMENT
+		COMMENT Checks for a custom nametag on living entities, and for teams/team visibilities for players.
```

Comment:
Should we mention the impl detail in the comment? At least add a `<p>` on it.

Reply:
I think mentioning the impl detail is fine as this is Minecraft code. The `<p>` would also be a good thing to add.